pom.xml

```diff
@@ -168,7 +168,7 @@
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.4</version>
+      <version>2.8.5</version>
     </dependency>
```

Comment:
I started using gson before knowing much about JSON parsers.
IIRC, Jackson has become the industry standard? e.g., doesn't Twitter use Jackson internally?
If so, we should probably switch over.
Probably not in this PR, but file an issue?

Reply:
Yes, twitter uses Jackson. 
Agreed.